ansible/roles_ocp_workloads/ocp4_workload_rhtr_pipelines/defaults/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+ocp_username: system:admin
+become_override: false
+silent: false
+
+ocp4_workload_rhtr_pipelines_dev_project_name: petclinic-dev
+ocp4_workload_rhtr_pipelines_prod_project_name: petclinic-prod
+ocp4_workload_rhtr_pipelines_jenkins_project_name: jenkins
+
+# The Source Code Repo to be set up in Jenkins (replace with actual user/password combination)
+ocp4_workload_rhtr_pipelines_gitea_user: gitea_user
+ocp4_workload_rhtr_pipelines_gitea_password: gitea_password
+ocp4_workload_rhtr_pipelines_gitea_repo: >-
+  http://gitea.gitea.svc:3000/{{ ocp4_workload_rhtr_pipelines_gitea_user }}/spring-petclinic
```

Comment:
Looks like something has gone wrong here... probably because the repo has changed and this role was removed.